dev/PushNotifications/PushBackgroundTaskInstance.h

```diff
@@ -1,26 +1,28 @@
-// Copyright (c) Microsoft Corporation and Contributors.
+﻿// Copyright (c) Microsoft Corporation and Contributors.
 // Licensed under the MIT License.
 
 #include <winrt/Windows.ApplicationModel.background.h>
+#include "PushRawNotification.h"
+#include "PushNotificationDummyDeferral.h"
 
 // Mocks IBackgroundTaskInstance to send raw payloads to packaged apps in
 // PushNotificationBackgroundTask::Run by com activation from PushNotificationsLongRunningProcess
 struct PushBackgroundTaskInstance : winrt::implements<PushBackgroundTaskInstance, winrt::Windows::ApplicationModel::Background::IBackgroundTaskInstance>
```

Comment:
Isn't this a MockBackgroundTaskInstance technically speaking? If so, shouldn't we rename as such?

Reply:
This is only used for LRP->App. But I can rename to MockBackgroundTaskInstance instead